stylesheets/components/card.css

```diff
@@ -0,0 +1,131 @@
+@import url(/stylesheets/abstract/variables.css);
+@import url(/stylesheets/abstract/color.css);
+@import url(/Stylesheets/main.css);
+
+.container {
+  margin-top: 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 2rem;
+}
+.card {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--light-grey);
+  margin-left: 2rem;
+}
+.card-doc {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--light-grey);
+  margin-left: 1rem;
+}
+.cardbox {
+  box-shadow: 7px 12px var(--dark-grey);
+}
+.card-only {
+  height: 11.5rem;
+  box-shadow: 20, 20 var(--bg-blue);
```

Comment:
This seems confusing, check this once.
Also, the whole point of box-shadow is to not make it completely visible. Go lighter.

Reply:
ya i forgot to delete it. done